content/foundation/glossary.md

```diff
@@ -506,6 +574,20 @@ process](http://incubator.apache.org/incubation/Process_Description.html).
 
 </dd>
 
+<dt>PPMC  {#PPMC}</dt>
```

Comment:
FWIW I'm ok with adding this acronym, as we are already using it

Reply:
Thank you. Let me first narrow the scope of this PR to adding what we are already using.

Leave the M- ones for further discussion.